src/pages/TermsOfService.test.ts

```diff
@@ -15,11 +15,33 @@ jest.mock('react-notifications-component', () => {
   };
 });
 
+type NavExports = typeof import('src/libs/nav');
+jest.mock(
+  'src/libs/nav',
+  (): NavExports => ({
+    ...jest.requireActual('src/libs/nav'),
```

Comment:
`jest.requireActual` takes a type parameter.

```suggestion
    ...jest.requireActual<NavExports>('src/libs/nav'),
```